### ISSUE TITLE
GroupNormalization relation to Instance and LayerNormalization

### ISSUE THREAD

**Issue by CNacherC-UV** [keras-team/keras#18143]
In the [GroupNormalization Layer documentation](https://keras.io/api/layers/normalization_layers/group_normalization/) , it says the following:

> "Relation to Layer Normalization: If the number of groups is set to 1, then this operation becomes nearly identical to Layer Normalization (see Layer Normalization docs for details).
> 
> Relation to Instance Normalization: If the number of groups is set to the input dimension (number of groups is equal to number of channels), then this operation becomes identical to Instance Normalization."

isn't Layer swapped for Instance ? Actually LayerNormalization is defined as that normalisation across al features (channels)

Best regards

**Comment by tilakrayal** [comment]
@CNacherC-UV,
Layer Normalization the normalization happens across the axes within each example, rather than across different examples in the batch.
https://keras.io/api/layers/normalization_layers/layer_normalization/

Group Normalization divides the channels into groups and computes within each group the mean and variance for normalization. We can relate to both Layer Normalization and Instance Normalization with Group Normalization.
https://keras.io/api/layers/normalization_layers/group_normalization/
Thank you!
